### PR TITLE
Telemetry: Metric collection from all resources for Ceph driver with its mock test

### PR DESCRIPTION
**Pull request by Click2Cloud-Gamma**
**What this PR does / why we need it**:
The current Metric driver collects metrics from "pool" resources only. This PR will enable Ceph Metrics Driver to collect metrics from all other resources and also it supports new interface defined in https://github.com/opensds/opensds/pull/764 
Mock test for Ceph Metrics driver is also added with this PR.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:
This PR depends on PR https://github.com/opensds/opensds/pull/764 
**Release note**:

```release-note
```

### CODE REVIEW

**Comment by Click2Cloud-Gamma** [comment]
Since this PR depends on #764 , it will not pass travis check.

File: contrib/drivers/ceph/metrics_cli.go [hunk 1]
```diff
@@ -97,87 +244,673 @@ func (cli *MetricCli) CollectMetrics(metricList []string, instanceID string) ([]
...
+			fmt.Sprintf("%v", mon_stats.Quorum),
+			"", const_label,
+			"",
+			var_label,
+			""})
+	}
+	return returnMap, nil
+}
+
+func (cli *MetricCli) CollectMetrics() ([]CephMetricStats, [2]string, error) {
```
Comment (wisererik) [inline contrib/drivers/ceph/metrics_cli.go]: should define length in return values?

**Comment by Click2Cloud-Gamma** [comment]
@wisererik @NajmudheenCT @leonwanghui 
Please review, Thanks.

**Comment by leonwanghui** [comment]
@wisererik PTAL, thanks!

**Pull request merged**